aea/test_tools/test_skill.py

```diff
@@ -367,8 +364,8 @@ def prepare_skill_dialogue(
             message = self.build_incoming_message(
                 message_type=dialogues.message_class,
                 dialogue_reference=dialogue_reference,
-                message_id=message_id,
-                target=target,
+                message_id=Dialogue.STARTING_MESSAGE_ID,
+                target=target or 0,
```

Comment:
target cannot be None, it should be a default value otherwise

Reply:
done